packages/expect/src/__tests__/matchers.test.js

```diff
@@ -1699,7 +1698,7 @@ describe('.toHaveProperty()', () => {
     [{a: {b: {c: 5}}}, 'a.b', {c: 4}],
     [new Foo(), 'a', 'a'],
     [new Foo(), 'b', undefined],
-    // [{a: {}}, 'a.b', undefined], // add for breaking change in future major
+    [{a: {}}, 'a.b', undefined], // wait until Jest 25
```

Comment:
```suggestion
    [{a: {}}, 'a.b', undefined],
```

next release will be 25, so we don't need the comment